### PR TITLE
stylng page 

### PR DESCRIPTION
**Pull request by alexander16108**
### we added styling to the page to resemble the sketched out page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,8 +14,8 @@ <h1 class="title">Awesome Books Tracker</h1>
       <div id="collection" class="collection"></div>
     </section>
     <section id="add_book">
-      <h1 class="title">Add a new book</h1>
-      <form id="bookAdderForm">
+      <h2 class="title">Add a new book</h2>
```
Comment (voscarmv) [inline index.html:17]: - [ ] Plesae use semantic HTML here to denote the `<header>` in the document.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @alexander16108 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- Please fix the Live Demo link on your `README.md` file. It currently is broken and takes me to a `404` not found page.
- Please make your HTML more semantic as per the comments below.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A. Your project is surprisingly complete and I can't find any optional suggestions to make! Congrats!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:19]: * done 👍

**Comment by Davidosky007** [comment]
Hi @alexander16108 @ahzia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
good-job-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ahzia — APPROVED** [review]
Everything looks perfect, 👏🤩,
Let's merge to main

**Pull request merged**